.github/ISSUE_TEMPLATE/question.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Ask a question
-about: Please use the community forum (https://github.com/iTwin/appui/issues) for general questions about using iTwin.js AppUI.
+about: Please use the community forum(https://github.com/iTwin/appui/issues) for general questions about using iTwin.js AppUI.
```

Comment:
Unintentional edit?
Could we replace this with https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser `contact_links` and direct to our `https://github.com/iTwin/appui/discussions` page instead?

Reply:
To clarify when you say "replace this" you mean updating the `config` to the template you provided and moving the `about` value to that file?

Reply:
also yes that edit was unintentional

Reply:
Yes, let's get rid of this template and add the about section to the `config.yml` :)